### DISCUSSION TITLE
Unwanted padding on ListViewItem in .net MAUI

### DISCUSSION THREAD

**Discussion by Laradius** [dotnet/maui#8562] [Q&A]
I have created a ListView which is binded to desired items.

However, on each ListViewItem there is unwanted padding which i have checked in XAML visual preview (12,0,0,0)

However, I haven't found any way to set the padding to 0 for ListViewItem or ViewCell as they don't have those properties accessable. They are only accessable through xaml live visual preview.

Is there any way to remove the unwanted padding?

I'll include the picture for visualization (ListView has red background, and children have blue background)
[Image of problem](https://i.stack.imgur.com/45NT6.png)

Here is my code:

```
<ListView ItemsSource="{Binding Items}" Background="Red">
            <ListView.ItemTemplate>
                <DataTemplate>
                    <ViewCell>
                        <Grid RowDefinitions="*" ColumnDefinitions="*">
                            <Label Grid.Column="0" Grid.Row="0" BackgroundColor="Blue" HorizontalTextAlignment="Start" HorizontalOptions="Start" Text="Hello World"/>
                        </Grid>
                    </ViewCell>
                </DataTemplate>
            </ListView.ItemTemplate>
        </ListView>
```

**Comment by jsuarezruiz** [comment]
Will be fixed in the next release by https://github.com/dotnet/maui/pull/8584
